MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
+            following steps:
+            <ol>
+            <li>Let <var>p</var> be a promise.</li>
+            <li>Run the following steps in parallel:
+             <ol>
+             <li>Let <var>resultList</var> be an empty list.
+             <li>For each <a href="https://tools.ietf.org/html/rfc2046">MIME
```

Comment:
This needs to say something about normalization, presumably there are an infinite number of strings for which `isSupportedType(mimeType)` returns true because of whitespace?

Also needs to say something about codecs, for example should both `video/webm` and `video/webm; codecs=vorbis,vp8` be returned, or which set of string?